rcsb/utils/dictionary/DictMethodCompModelHelper.py

```diff
@@ -333,13 +333,20 @@ def consolidateGlobalQAScores(self, dataContainer, catName, **kwargs):
             vD = {}
             mD = {}
 
+            # Get representative model
+            repModelId = self.__commonU.getRepresentativeModelId(dataContainer)
```

Comment:
@piehld https://github.com/rcsb/py-rcsb_utils_dictionary/blob/ro-4455/rcsb/utils/dictionary/DictMethodCommonUtils.py#L4589 and https://github.com/rcsb/py-rcsb_utils_dictionary/blob/ro-4455/rcsb/utils/dictionary/DictMethodCommonUtils.py#L4506 do not handle representative models for computed structure models. See https://mmcif.wwpdb.org/dictionaries/mmcif_ma.dic/Categories/ma_model_representative.html.

Reply:
I think it would be ok to handle the consolidation of selecting representative models for CSMs separately.

Reply:
Good point. Yes, for CSMs it currently just defaults to the first model ID ('1'), so we should add explicit selection for that. We'll also need to add explicit selection for other non-NMR method types. E.g., `_pdbx_soln_scatter_model.representative_conformer` for solution scattering (e.g., https://www.rcsb.org/structure/1HAQ).